py3status/modules/yandexdisk_status.py

```diff
@@ -46,17 +47,17 @@ class Py3status:
     status_off = 'Not started'
     status_on = 'Idle'
 
+    def post_config_hook(self):
+        if not self.py3.check_commands('yandex-disk'):
+            raise Exception(STRING_NOT_INSTALLED)
+
     def yandexdisk(self):
-        if not self.py3.check_commands(["yandex-disk"]):
-            return {'cached_until': self.py3.CACHE_FOREVER,
-                    'color': self.py3.COLOR_BAD,
-                    'full_text': string_unavailable}
         try:
             status = self.py3.command_output('yandex-disk status').splitlines()[0]
         except:
             return {'cache_until': self.py3.CACHE_FOREVER,
                     'color': self.py3.COLOR_ERROR or self.py3.COLOR_BAD,
-                    'full_text': string_error}
+                    'full_text': STRING_ERROR}
```

Comment:
Here we should use `self.py3.error(STRING_ERROR)` then user can hide error etc.  The string will need updating.